app/controllers/api/v3/timelines/channels_controller.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 class Api::V3::Timelines::ChannelsController < Api::BaseController
-  before_action :set_channel
```

Comment:
Is this intentional?

Reply:
nope. my bad, i remove this to circumvent having to lookup on acctrelay locally.